README.md

```diff
@@ -12,7 +12,7 @@
 
 ## Live Demo (if available)
 
-[coming soon](https://livedemo.com)
+https://habtamu210.github.io/portfoliopage/
```

Comment:
[OPTIONAL] I would recommend you use this format `[online version](your url)` so that your readme would look professional.